tests/testthat/test-shinytest2-tm_file_viewer.R

```diff
@@ -0,0 +1,11 @@
+test_that("e2e: tm_file_viewer initializes without errors and shows encodings", {
```

Comment:
Let's test whether tree lists files specified in the argument.
```suggestion
test_that("e2e: tm_file_viewer initializes without errors and shows files tree specified in input_path argument", {
```